examples/java7/build-java7.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -e
+
+which acbuild || (echo "Make sure acbuild is on your path"; exit 1)
+
+TMPDIR=$(mktemp -d)
+PACKAGESDIR=$(mktemp -d)
+
+rmDirectories() {
+    rm -rf ${TMPDIR}
+    rm -rf ${PACKAGESDIR}
+}
+trap rmDirectories EXIT
+
+pushd ${PACKAGESDIR}
+apt-get download \
```

Comment:
Well it came from the giant unordered list of packages that was a separate file before. Think if I kept this in a separate file still it would be a little saner?